src/traces/histogram2d/attributes.js

```diff
@@ -0,0 +1,50 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+var histogramAttrs = require('../histogram/attributes');
+var heatmapAttrs = require('../heatmap/attributes');
+
+module.exports = {
+    x: histogramAttrs.x,
+    y: histogramAttrs.y,
+
+    z: {
+        valType: 'data_array',
+        description: 'Sets the aggregation data.'
```

Comment:
@chriddyp @cldougl fixes an old plot schema bug that put `z` in `histogram` erroneously.

Now each trace type has its own `attributes` object. `histogram2d` and `histogram2dcontour` have `z`, but not `histogram`.